maps.cpp

```diff
@@ -0,0 +1,42 @@
+#include <unordered_map>
+#include <string>
+#include "maps.h"
+using namespace std;
+const unordered_map<int, int> mounthDay = {

```

Comment:
Are you sure `const` is enough and no body can change the content of this map at runtime?

Reply:
yes it's enough